nibe/connection/nibegw.py

```diff
@@ -267,6 +317,21 @@ def _on_raw_coil_value(self, coil_address: int, raw_value: bytes):
         logger.info(f"{coil.name}: {coil.value}")
         self._heatpump.notify_coil_update(coil)
 
+    def _on_coil_value(self, coil_address: int, value: Union[float, int, str]):
```

Comment:
In what conditon it can be a string?

Reply:
It can't here, more for completeness. Secondly, value = get_mapping_for(value..) would be a type violation without a intermediary variable since it returns str.